hooks/hookstage/entrypoint.go

```diff
@@ -8,7 +8,7 @@ import (
 type Entrypoint interface {
 	HandleEntrypointHook(
 		context.Context,
-		InvocationContext,
+		*ModuleContext,
```

Comment:
Yes, we consider it to be safe because a module can implement a hook stage only once, hence the event described by you `in the current stage multiple hooks for a given module` is not possible.

Reply:
UPD: after all we decided to remove the pointer and rewrite a bit the logic for module contexts, because we faced some issues with concurrency for the next stages BidderRequest and RawBidderResponse since each bidder is processed in a separate go routine